ontology/yaml/resources/SAFETY/entity_types/FD.yaml

```diff
@@ -19,4 +19,11 @@ FD_STD:
   implements:
   - FD
   uses:
-  - damper_status
\ No newline at end of file
+  - damper_status
+  opt_uses:
+  - supply_air_damper_open_status
```

Comment:
if there is a supply air damper on the same device, then what is the damper above? either remove these from this FD or differentiate the field above.

Reply:
This field was already existing. Did not want to break the existing field, hence modeled it in opt_uses with both supply and return dampers.

Reply:
The field was already existing. Did not want to break existing model, hence modeled under opt_uses for both supply and return dampers

Reply:
but the supply and return on what? typically a fire smoke damper is its own stand-alone damper, and there is no reference to supply or return air.

Reply:
i think you are modeling too many dampers as part of the FD.

Reply:
Noted, we will close this pull request and model a generic FD with two dampers as a separate pull request.